src/metabase/models/permissions.clj

```diff
@@ -487,12 +487,24 @@
   permissions path for the table. Otherwise, defaults to the root path ('/'), thus restricting writes to admins."
   [db-id schema table-id]
   (let [f (u/ignore-exceptions
-           (classloader/require ' metabase-enterprise.advanced-permissions.models.permissions)
-           (resolve ' metabase-enterprise.advanced-permissions.models.permissions/data-model-write-perms-path))]
-    (if (and f premium-features/enable-advanced-permissions?)
+           (classloader/require 'metabase-enterprise.advanced-permissions.models.permissions)
+           (resolve 'metabase-enterprise.advanced-permissions.models.permissions/data-model-write-perms-path))]
+    (if (and f (premium-features/enable-advanced-permissions?))
```

Comment:
🤦 this needed to be a function call. added a couple tests that would have caught this.